src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -813,7 +838,7 @@ def iot_hub_get_stats(client, hub_name, resource_group_name=None):
 
 
 def validate_authentication_type_input(endpoint_type, connection_string=None, authentication_type=None, endpoint_uri=None, entity_path=None):
-    is_keyBased = (AuthenticationType.KeyBased.value == authentication_type.lower()) or (authentication_type is None)
+    is_keyBased = (AuthenticationType.KeyBased.value == authentication_type) or (authentication_type is None)
```

Comment:
Why is case sensitive when comparing here?

Reply:
It's not - `authentication_type` has been changed in inputs to be an enum of `AuthenticationType` - and the constant was changed to match the swagger / API value - no more need to convert to `lower()` since they'll be using the enum / constant value.

Reply:
Got it, thanks~